lib/commanded/event/handler.ex

```diff
@@ -626,6 +626,15 @@ defmodule Commanded.Event.Handler do
         end)
 
         handle_event_error(error, event, state, context)
+
+      {:error, reason, stacktrace} ->
+        Logger.error(fn ->
+          describe(state) <>
+            " failed to handle event #{inspect(event)} due to: #{inspect(reason)}"
```

Comment:
In the future, I would suggest we move to the Exception formatter to log these. I can open another issue if we think that's a good idea.

Reply:
I added the `pretty` option to inspect the error messages. It makes the log much easier to read.

<img width="1072" alt="Screen Shot 2020-02-17 at 7 26 21 AM" src="https://user-images.githubusercontent.com/1019721/74653866-0ff5e800-5157-11ea-9a3f-d965175b79fc.png">